durabletask-azuremanaged/durabletask/azuremanaged/durabletask_grpc_interceptor.py

```diff
@@ -3,28 +3,23 @@
 
 from durabletask.internal.grpc_interceptor import _ClientCallDetails, DefaultClientInterceptorImpl
 from durabletask.azuremanaged.internal.access_token_manager import AccessTokenManager
-
+from azure.core.credentials import TokenCredential
 import grpc
 
 class DTSDefaultClientInterceptorImpl (DefaultClientInterceptorImpl):
     """The class implements a UnaryUnaryClientInterceptor, UnaryStreamClientInterceptor,
     StreamUnaryClientInterceptor and StreamStreamClientInterceptor from grpc to add an 
     interceptor to add additional headers to all calls as needed."""
 
-    def __init__(self, metadata: list[tuple[str, str]]):
+    def __init__(self, token_credential: TokenCredential, taskhub_name: str):
+        metadata = [("taskhub", taskhub_name)]

```

Comment:
Looks like this variable is assigned but not used? Did you mean to use `_metadata` here?